app/src/main/java/com/orange/ouds/app/ui/utilities/composable/CustomizationElements.kt

```diff
@@ -89,4 +90,24 @@ fun CustomizationChoiceChipsColumn(
             }
         }
     }
+}
+
+@Composable
+fun CustomizationTextFieldColumn(
```

Comment:
As discussed quickly the other day, I think we should only add suffix like `Box`, `Row` or `Column` when the composable is actually a container where we can add content (for instance `OudsColoredBox`). If we finally choose to change that kind of naming, this can also be applied to `CustomizationChoiceChipsColumn`. However I would say that we can keep `CustomizationSwitchListItem` because it is actually a `ListItem`.